packages/muelu/test/unit_tests_kokkos/Utilities_kokkos.cpp

```diff
@@ -77,25 +100,551 @@ namespace MueLuTests {
     RCP<MultiVector> nullspace = MultiVectorFactory::Build(map, 1);
     nullspace->putScalar(Teuchos::ScalarTraits<SC>::one());
 
-    MueLu::Utilities_kokkos<SC,LO,GO,NO>::Transpose(*A);// compile test
+    MueLu::Utilities_kokkos<SC, LO, GO, NO>::Transpose(*A); // compile test
 
     // CM Test
-    RCP<Xpetra::Vector<LocalOrdinal,LocalOrdinal,GlobalOrdinal,Node> > ordering
-      = MueLu::Utilities_kokkos<SC,LO,GO,NO>::CuthillMcKee(*A);
+    RCP<Xpetra::Vector<LocalOrdinal, LocalOrdinal, GlobalOrdinal, Node>> ordering = MueLu::Utilities_kokkos<SC, LO, GO, NO>::CuthillMcKee(*A);
 
+    RCP<Xpetra::Vector<LocalOrdinal, LocalOrdinal, GlobalOrdinal, Node>> ordering2 = MueLu::Utilities_kokkos<SC, LO, GO, NO>::ReverseCuthillMcKee(*A);
 
-    RCP<Xpetra::Vector<LocalOrdinal,LocalOrdinal,GlobalOrdinal,Node> > ordering2
-      = MueLu::Utilities_kokkos<SC,LO,GO,NO>::ReverseCuthillMcKee(*A);
+    TEST_EQUALITY(1, 1);
```

Comment:
?

Reply:
This is actually a leftover from original file (only formatting has changed for CuthillMcKee test). I should probably change that, totally forgot about this test